src/main/java/CLIOutput.java

```diff
@@ -1,16 +1,13 @@
-import java.util.*;
-import java.io.*;
-
-Class CLIOutput {
+public class CLIOutput {
     public static void outputBoard (Board board, boolean symbol) {
         // Prints the board to the console
         
-        Pieces pieceList = board.getPieces();
+        Piece[][] pieceArray = board.getBoard();
         String[][] outputBoard = new String[8][8];
```

Comment:
8 and 8 are [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) and should be replaced with named variables to clarify that they are width and height. Plus, board dimensions are already fixed in the board class, so it would be best to reference those.

Reply:
Seems unnecessary to create a String array; wouldn't a char array work fine?